README.md

```diff
@@ -57,10 +38,134 @@ The following commands are for Ubuntu Server. However, equivalent commands shoul
    * Debug Build: `cargo build`
    * Release Build: `cargo build --release`
    * Run Tests: `cargo test`
-   * Run Server: `cargo run --bin modelardbd path_to_data_folder`
+   * Run Server: `cargo run --bin modelardbd path_to_local_data_folder`
    * Run Client: `cargo run --bin modelardb [server_address] [query_file]`
 5. Move `modelardbd` and `modelardb` from the `target` directory to any directory.
 
+## Usage
+`modelardbd` supports two execution modes *edge* and *cloud*. For storage, `modelardbd` uses local storage and an Amazon S3-compatible object store (optional in edge mode). The execution mode dictates where queries are executed, when `modelardbd` is deployed in edge mode it executes queries against local storage and when it is deployed in cloud mode it executes queries against the object store. For both deployment modes, `modelardbd` automatically compresses the ingested time series using multiple different types of models and continuously transfers this compressed representation from local storage to the object store. Be aware that sharing metadata between multiple instances of `modelardbd` is currently under development, thus only the instance of `modelardbd` that ingested a time series can currently query it.
+
+### Start Server
+To run `modelardbd` in edge mode using only local storage, i.e., without transferring the ingested time series to an object store, simply pass the path to the local folder `modelardbd` should use as its data folder:
+
+```shell
+modelardbd edge path_to_local_data_folder
+```
+
+To automatically transfer ingested time series to an object store the following environment variable must first be set to appropriate values so `modelardbd` knows which object store to connect to, how to authenticate, and if an HTTP connection is allowed or if HTTPS is required:
+
+```shell
+AWS_ACCESS_KEY_ID
+AWS_SECRET_ACCESS_KEY
+AWS_DEFAULT_REGION
+AWS_ENDPOINT
+AWS_ALLOW_HTTP
+```
+
+For example, to use a local instance of [Minio](https://min.io/), assuming the access key id `KURo1eQeMhDeVsrz` and the secret access key `sp7TDyD2ZruJ0VdFHmkacT1Y90PVPF7p`has been created through [Minio's web interface](http://127.0.0.1:9001/access-keys), set the environment variable as follows:
```

Comment:
Missing space after the secret access key.

Reply:
Fixed, and changed `environment variable` to `environment variables`.